homeassistant/components/roborock/image.py

```diff
@@ -98,6 +110,9 @@ async def async_image(self) -> bytes | None:
         if self.is_map_valid():
             map_data: bytes = await self.cloud_api.get_map_v1()
             self.cached_map = self._create_image(map_data)
+            await self._roborock_storage.async_save_map(
```

Comment:
Can we do this in a background task? We wouldn't want the user to wait to get the image because we're writing it to disk.

```python
self.entity_platform.config_entry.async_create_task(
    self._roborock_storage.async_save_map(…)
)
```